### PR TITLE
docs: add collection’s root option

### LINKED ISSUE #75: Document root option for collections
See frctl/fractal#642

### PR DESCRIPTION
**Pull request by LeBenLeBen**
closes #75

### CODE REVIEW

File: docs/guide/collections/configuration-reference.md [hunk 1]
```diff
@@ -44,6 +44,14 @@ The string that is used when a UI needs a title for the collection. Defaults to
 title: 'My Favourite Website Layouts'
 ```
 
+#### root <Badge text="added in v1.5.0" type="tip"/>
```
Comment (mihkeleidast) [inline docs/guide/collections/configuration-reference.md:47]: should we add a note that this also requires theme support? e.g. users with a custom theme / mandelbrot fork that do not update can't use it.

File: docs/guide/collections/configuration-reference.md [hunk 2]
```diff
@@ -44,6 +44,14 @@ The string that is used when a UI needs a title for the collection. Defaults to
...
+
+Pull the collection to the navigation top level, next to the defaults "Components" and "Documentation".
```
Comment (mihkeleidast) [inline docs/guide/collections/configuration-reference.md:49]: maybe worth mentioning that this only works for component collections, I believe I did not add support for doc collections nor did I test this.

**Review by mihkeleidast — CHANGES_REQUESTED** [review]

File: docs/guide/collections/configuration-reference.md (hunk 1 above)
Comment (LeBenLeBen) [reply docs/guide/collections/configuration-reference.md]: Indeed 👍

File: docs/guide/collections/configuration-reference.md (hunk 2 above)
Comment (LeBenLeBen) [reply docs/guide/collections/configuration-reference.md:49]: ✅

**Review by mihkeleidast — APPROVED** [review]

**Pull request merged**